src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -116,25 +116,11 @@ private static void printHelp() throws IOException {
         sb.append("============================\n");
         sb.append("Please provide the directory path to a configuration file!");
         sb.append("\n");
-        sb.append("See: https://github.com/fcrepo4-labs/migration-utils/blob/master/");
-        sb.append("src/main/resources/spring/migration-bean.xml");
+        sb.append("For an example, see: https://github.com/fcrepo4-labs/migration-utils/blob/master/");
+        sb.append("src/test/resources/spring/migration-bean.xml");
         sb.append("\n\n");
-        sb.append("The configuration file should contain the following (with appropriate values):");
+        sb.append("For detailed instructions, see: https://github.com/fcrepo4-labs/migration-utils/blob/master/README.md");
         sb.append("\n");
-        sb.append("~~~~~~~~~~~~~~\n");
```

Comment:
You should be able to copy the test resources into the jar artifact with something along the lines of:
https://maven.apache.org/plugins/maven-resources-plugin/plugin-info.html

Let me know if you need some help.